src/pipecat/utils/asyncio.py

```diff
@@ -181,15 +264,72 @@ async def cancel_task(self, task: asyncio.Task, timeout: Optional[float] = None)
 
     def current_tasks(self) -> Sequence[asyncio.Task]:
         """Returns the list of currently created/registered tasks."""
-        return list(self._tasks.values())
+        return [data.task for data in self._tasks.values()]
 
-    def _add_task(self, task: asyncio.Task):
+    def start_watchdog(self, task: asyncio.Task):
+        """Starts the given task watchdog timer. If not reset, a warning will be
+        logged indicating the task is stalling.
```

Comment:
We are just showing a warning in case the task doesn't exist. Should we add some validation, or just update this comment ?

Reply:
Because I guess it could make sense to check if we have already started the watchdog and print a warning in this case.